includes/class-wc-admin-reports-sync.php

```diff
@@ -187,6 +189,7 @@ public static function is_importing() {
 				'status'   => 'pending',
 				'per_page' => 1,
 				'claimed'  => false,
+				'search'   => 'import',
```

Comment:
That was required, otherwise `wc_admin_unsnooze_admin_notes` was making `is_importing` to return true.